python/lsst/ts/criopy/gui/actuatorsdisplay/mirror_view.py

```diff
@@ -181,6 +195,33 @@ def update_fcu(self, fcu: FCUData, value: float, state: DataItemState) -> None:
         if self._selected_actuator.actuator.index == fcu.index:
             self.selectionChanged.emit(self._selected_actuator if self._selected_actuator.active else None)
 
+    def update_scanner(self, tc_values: list[tuple[ThermocoupleData, float]], state: DataItemState) -> None:
+        """
+        Update Scanner's value and state.
+
+        Parameters
+        ----------
+        """
```

Comment:
More detail of doc string in `Parameters`?

Reply:
Added. Thanks.